daisy_workflows/image_import/windows/translate.ps1

```diff
@@ -265,25 +268,28 @@ function Install-32bitPackages {
 
 try {
   Write-Output 'Translate: Beginning translate PowerShell script.'
-  $script:outs_dir = Get-MetadataValue -key 'daisy-outs-path'
-  $script:install_packages = Get-MetadataValue -key 'install-gce-packages'
-  $script:sysprep = Get-MetadataValue -key 'sysprep'
-  $script:is_byol = Get-MetadataValue -key 'is_byol'
-  $script:is_x86 = Get-MetadataValue -key 'is_x86'
 
   Remove-VMWareTools
   Change-InstanceProperties
   Configure-Network
   Setup-NTP
   Configure-RDPSecurity
 
-  if ($script:is_x86.ToLower() -ne 'true') {
+  $script:install_packages = Get-MetadataValue -key 'install-gce-packages'
+  $script:sysprep = Get-MetadataValue -key 'sysprep'
+  $script:is_byol = Get-MetadataValue -key 'is_byol'
+  if ($script:install_packages -eq $null -or $script:sysprep -eq $null -or $script:is_byol -eq $null) {
+    Write-Output "Translate: failed to obtain at least one of the required values from metadata, rebooting in an attempt to resolve issue. install_packages=$script:install_packages, sysprep=$script:sysprep, is_byol=$script:is_byol"
+    Restart-Computer -Force
+  }
```

Comment:
Any idea why a reboot helps? Thoughts on limiting the number of reboots allowed before quitting?

Reply:
Limiting to one reboot is pretty reasonable. Since the startup scripts use the temp directory, I'll add a file exist check and creation of the file in the temp directory to apply this limit.

This is a non-scientific answer, but in Windows especially when dealing with driver changes a reboot goes a long way to fixing many things. It's also better than this being a fatal error where the entire import is aborted.

Reply:
I dug into this further and realized that at least one of the failure had been after removing vmware tools which also reboots the system. Which got me thinking that in these situations the script was starting right at boot instead of after setupcomplete.cmd, so also added the 60 second delay the other scripts have to help deal with timing issues.

Reply:
sgtm (I see that we're doing the `ping 127.0.0.1 -n 60` pattern in several other Windows startup scripts in this repo). It sounds like there's an opportunity to investigate the root cause of why `ping 127.0.0.1 -n 60` helps, but that's probably beyond the scope of this PR.